frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -177,10 +187,56 @@ ListDropdown.propTypes = {
 
 const NsDropdown_ = (props) => {
   const openshiftFlag = props.flags[FLAGS.OPENSHIFT];
+
   if (flagPending(openshiftFlag)) {
     return null;
   }
-  const kind = openshiftFlag ? 'Project' : 'Namespace';
+
+  const nsModel = modelFor('Namespace');
+  const useNamespaceAccessReview = () => {
+    const canCreateNamespace = useAccessReview({
+      group: nsModel.apiGroup,
+      resource: nsModel.plural,
+      verb: 'create',
+    });
+    return canCreateNamespace;
+  };
```

Comment:
It's unclear to me why this is wrapped in a function. It will always be truthy in the checks below. Shouldn't this just be

```suggestion
  const canCreateNamespace = useAccessReview({
    group: nsModel.apiGroup,
    resource: nsModel.plural,
    verb: 'create',
  });
```

Reply:
The eslint was complaining: 
```
error  React Hook "useAccessReview" is called conditionally. React Hooks must be called in the exact same order in every component render 
```

Reply:
> The eslint was complaining:
> 
> ```
> error  React Hook "useAccessReview" is called conditionally. React Hooks must be called in the exact same order in every component render 
> ```

I think your solution to wrap it in a function has the same problem. You're just hiding it from eslint :)

You need to move the hook above the OpenShift flag check so it always run.

Reply:
handled this